CHANGELOG.md

```diff
@@ -130,81 +107,102 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Bump `actions/setup-go` from 4 to 5 ([#1696](https://github.com/NibiruChain/nibiru/pull/1696))
 - Bump `golang` from 1.19 to 1.21 ([#1698](https://github.com/NibiruChain/nibiru/pull/1698))
 - [#1678](https://github.com/NibiruChain/nibiru/pull/1678) - chore(deps): collections to v0.4.0 for math.Int value encoder
-- [#1809](https://github.com/NibiruChain/nibiru/pull/1809) - chore: bump grocksdb to v1.8.12
-- [#1812](https://github.com/NibiruChain/nibiru/pull/1812) - chore(deps): bump `cosmos-sdk` to v0.47.10
 
-## [v1.1.0] - 2023-11-20
+## v1.1.0
 
-- [[Release Link](https://github.com/NibiruChain/nibiru/releases/tag/v1.1.0)]
-  [[Commits](https://github.com/NibiruChain/nibiru/commits/v1.1.0)]
+### Features
 
-### State Machine Breaking
+- [#1670](https://github.com/NibiruChain/nibiru/pull/1670) - feat(inflation): Make inflation polynomial
+- [#1682](https://github.com/NibiruChain/nibiru/pull/1682) - feat!: add upgrade handler for v1.1.0
+- [#1776](https://github.com/NibiruChain/nibiru/pull/1776) - feat(inflation): make inflation params a collection and add commands to update them
+- [#1795](https://github.com/NibiruChain/nibiru/pull/1795) - feat(inflation): add inflation tx cmds
 
-- [#1670](https://github.com/NibiruChain/nibiru/pull/1670) - feat(inflation)!: Make inflation follow polynomial distribution from the tokenomics
+### Bug Fixes
 
-## [v1.0.0] - 2023-11-07
+- [#1688](https://github.com/NibiruChain/nibiru/pull/1688) - fix(inflation): make default inflation allocation follow tokenomics
+- [#1706](https://github.com/NibiruChain/nibiru/pull/706) - fix: `v1.1.0` upgrade handler
+- [#1786](https://github.com/NibiruChain/nibiru/pull/1786) - fix(inflation): fix inflation off-by 2 error
+- [#1796](https://github.com/NibiruChain/nibiru/pull/1796) - fix(inflation): fix num skipped epoch when inflation is added to an existing chain
+- [#1797](https://github.com/NibiruChain/nibiru/pull/1797) - fix(inflation): fix num skipped epoch updates logic
+- [#1804](https://github.com/NibiruChain/nibiru/pull/1804) - fix(inflation): update default parameters
 
```

Comment:
It is generally more important to know if a change is state machine breaking than to classify it as feat versus fix because that's handled by the conventional commit prefix.